lib/time.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Pairmotron.Time do
+
+  @doc """
+  Returns true if the year and week passed in are the same week as the current
+  date. Weeks start on Monday. The first week of the year is defined as the first
+  week that contains a Thursday.
+  """
+  def same_week?(year, week, current_date) do
```

Comment:
Did some thinking on this. It doesn't make a huge difference really. Making a function have less parameters is always a good thing in a vacuum, but it makes calling it look a little more complicated (not a strong argument there). The thing I like most about making it two parameters, is that it makes both parameters more similar to each other. They're still not identical, since the second parameter is actually a date object, but they are more similar.